### PR TITLE
Room name warning

### PR DESCRIPTION
**Pull request by zbettenbuk**
image

image

image

image

### CODE REVIEW

File: react/features/base/label/components/CircularLabel.native.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import AbstractCircularLabel, {
     type Props as AbstractProps
 } from './AbstractCircularLabel';
 import styles from './styles';
+import { Icon } from '../../icons';
```
Comment (saghul) [inline react/features/base/label/components/CircularLabel.native.js:11]: sorting

File: react/features/conference/components/native/Labels.js [hunk 2]
```diff
@@ -200,14 +204,18 @@ class Labels extends AbstractLabels<Props, State> {
...
                     </TouchableOpacity>
-                    {/*
-                      * Emil, Lyubomir, Nichole, and Zoli said that the Labels
-                      * should not be rendered in Picture-in-Picture. Saul
-                      * argued that the recording Labels should be rendered. As
-                      * a temporary compromise, don't render the
-                      * VideoQualityLabel at least because it's not that
-                      * important.
-                      */
+                    <TouchableOpacity
```
Comment (saghul) [inline react/features/conference/components/native/Labels.js:207]: have you checked how it looks slike in PiP mode?

File: react/features/conference/components/native/styles.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { ColorSchemeRegistry, schemeColor } from '../../../base/color-scheme';
 import { FILMSTRIP_SIZE } from '../../../filmstrip';
 
 export const NAVBAR_GRADIENT_COLORS = [ '#000000FF', '#00000000' ];
+export const INSECURE_ROOM_NAME_LABEL_COLOR = ColorPalette.warning;
```
Comment (saghul) [inline react/features/conference/components/native/styles.js:6]: why the constant if it's only used once?

File: react/features/e2ee/components/E2EELabel.js [hunk 4]
```diff
@@ -29,16 +29,13 @@ class E2EELabel extends Component<Props> {
             return null;
         }
 
-        // eslint-disable-next-line react/jsx-max-props-per-line
-        const icon = <Icon size = { 22 } src = { IconE2EE } />;
```
Comment (saghul) [inline react/features/e2ee/components/E2EELabel.js:33]: how about the size now?

File: react/features/welcome/components/AbstractWelcomePage.js [hunk 5]
```diff
@@ -78,6 +79,7 @@ export class AbstractWelcomePage extends Component<Props, *> {
         joining: false,
         room: '',
         roomPlaceholder: '',
+        insecureRoomName: false,
```
Comment (saghul) [inline react/features/welcome/components/AbstractWelcomePage.js:82]: sorting

File: react/features/e2ee/components/E2EELabel.js (hunk 4 above)
Comment (zbettenbuk) [reply react/features/e2ee/components/E2EELabel.js:33]: It now uses the default size, and I think that looks great.

File: react/features/conference/components/native/styles.js (hunk 3 above)
Comment (zbettenbuk) [reply react/features/conference/components/native/styles.js:6]: It's exported and being used by the expanded label component

Comment (saghul) [reply react/features/conference/components/native/styles.js:6]: Ah, my bad!

File: react/features/conference/components/native/Labels.js (hunk 2 above)
Comment (zbettenbuk) [reply react/features/conference/components/native/Labels.js:207]: Labels are not visible on the reduced UI.

Now with this said, i removes the unused reduced UI logic from the Labels component too

Comment (saghul) [reply react/features/conference/components/native/Labels.js:207]: The recording one is, that'ss what the (overly verbose) comment was about...

**Comment by saghul** [comment]
Looks like the bundle size exploed by quite a bit. I wonder if it's the new library.

**Comment by zbettenbuk** [comment]
I'm already looking into it

**Comment by pikurasa** [comment]
I just saw this feature, and think it is great.

It would be nice if, at least for desktop, it gave you a recommended (secure) room name, and maybe a link to that new recommended room.

**Comment by pandel** [comment]
Nice feature - for Internet-accessible installations, but somewhat irritating  if used in controlled and closed environments with auto-generated room names. Please consider a possibility to send a parameter via JitsiMeetConferenceOptions to deactivate this in a later revision of this feature.

For now, how can this be avoided when using the SDK? While I am compiling the SDK myself just tell me the line of code to comment out :-)

**Comment by c-prompt** [comment]
> EDIT: Found. Just always return false in react/features/base/util/isInsecureRoomName.js (dirty hack, but works for now)

I'm not thrilled with this feature either as I also have auto-generated names in addition to user tokens. Feature request would be to make this configurable in `/usr/share/jitsi-meet/interface_config.js` but another dirty hack:

Search in `/usr/share/jitsi-meet/css/all.css` for:
`.large-video-labels .circular-label.insecure{background:#d77976}`
Replace with:
`.large-video-labels .circular-label.insecure{display:none!important;background:#d77976}`

**Comment by mikweb2017** [comment]
I have servers where I have to manage jitsi this function do you not like how I can turn it off?

**Comment by luixxiul** [comment]
Hi guys, as PR is not an issue tracker, please open a new issue for a new problem. If you want to suggest that it be related to this PR, you could link from the issue simply by mentioning the number of this PR, which is #6754. Also, please do not forget to search for existing issues as your issue might be already reported. Thanks! 😄

**Comment by pandel** [comment]
{ot}This is no "issue", but a PR with missing but necessary control over its impact. It is so simple: think about embedded usage of the SDK and that's all. Should have been obvious from the start but wasn't actually.

Now all non-Internet or SDK users have to live with this "feature" as long as the issue isn't recognized as such. So, why not ask the developer here in its PR to add control to this "feature" and make it complete?{/ot}

**Comment by luixxiul** [comment]
> a PR with missing but necessary control over its impact. It is so simple.

> why not ask the developer here in its PR to add control to his "feature" and make it complete?{/ot}

I mean it is important to have an open ticket to track a bug. This PR has been closed and it is often that closed PR or issue will be overlooked as it is hidden from the tracker by default.

Thank you for leaving a comment on https://github.com/jitsi/jitsi-meet/issues/6801#issuecomment-635905233.

Just out of my experience, I would open a new ticket to track an issue for the merged PR like: https://github.com/jitsi/jitsi-meet/issues/6802

**Pull request merged**